lte/gateway/python/magma/redirectd/tests/BUILD.bazel

```diff
@@ -0,0 +1,33 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@python_deps//:requirements.bzl", "requirement")
+load("//bazel:python_test.bzl", "pytest_test")
+
+COMMON_TEST_DEPS = [
+    "//lte/gateway/python/magma/redirectd:redirectd",
+    requirement("pytest"),
+    requirement("pytest-cov"),
+    requirement("coverage-lcov"),
```

Comment:
maybe we can pull these out to `pytest_test` if they're always needed for tests.

Reply:
Good idea, pushed!